HHS/Head-Start-TTADP
# [TTAHUB-1162] Fix objective file ordering

## Description of change
Previously, we used an index to attach a given file to a given objective. Other changes have revealed the weakness of that approach. Because of #1152, we can now depend on an ID being present and find the objective data that way.

## How to test
- Add a new goal
- Add a new objective
- Add a second new objective
- Save draft
- Add files to second objective
- Add files to first objective

Confirm all is as it should be.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1162


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
